app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,76 @@
  *= require_tree .
  *= require_self
  */
+html, body {
```

Comment:
Please order alphabetically inside each definition and use rem instead of px by default (you can override the default rem size to 10 px with body{font-size:62.5%} for simpler maths)